server/models/migrations/next/migrate.sql

```diff
@@ -41,3 +41,60 @@ CALL add_column_if_missing('general_ledger', 'principal_center_id', 'MEDIUMINT(8
 
 CALL add_constraint_if_missing('general_ledger', 'general_ledger__cost_center_1', 'FOREIGN KEY (`cost_center_id`) REFERENCES `fee_center` (`id`) ON UPDATE CASCADE');
 CALL add_constraint_if_missing('general_ledger', 'general_ledger__cost_center_2', 'FOREIGN KEY (`principal_center_id`) REFERENCES `fee_center` (`id`) ON UPDATE CASCADE');
+
+
+/*
+ * @author: mbayopanda
+ * @date: 2021-09-02
+ * @desc: fee center report tables and test data
+ */
+CALL add_column_if_missing('fee_center', 'step_order', 'SMALLINT(5) NOT NULL DEFAULT 0');
+CALL add_column_if_missing('fee_center', 'default_fee_center_index_id', 'MEDIUMINT(8) UNSIGNED NULL');
+
+CREATE TABLE IF NOT EXISTS `fee_center_index` (
+  `id` MEDIUMINT(8) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `label` VARCHAR(100) NOT NULL,
+  `constant` TINYINT(1) DEFAULT 0,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
+
+CREATE TABLE IF NOT EXISTS `fee_center_index_value` (
+  `fee_center_index_id` MEDIUMINT(8) UNSIGNED NOT NULL,
+  `fee_center_id` MEDIUMINT(8) UNSIGNED NOT NULL,
+  `value` decimal(19,4) UNSIGNED DEFAULT NULL,
+  KEY `fee_center_index_id` (`fee_center_index_id`),
+  KEY `fee_center_id` (`fee_center_id`),
+  CONSTRAINT `fc_index_value__fee_center_index_id`FOREIGN KEY (`fee_center_index_id`) REFERENCES `fee_center_index` (`id`),
+  CONSTRAINT `fc_index__fee_center`FOREIGN KEY (`fee_center_id`) REFERENCES `fee_center` (`id`)
+) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
+
+INSERT INTO `fee_center_index` (`id`, `label`) VALUES 
```

Comment:
Please add INSERT IGNORE into all of these 'INSERTS' to make the script more robust.